tensorzero-internal/src/endpoints/openai_compatible.rs

```diff
@@ -296,10 +296,41 @@ pub struct OpenAICompatibleParams {
     tensorzero_tags: HashMap<String, String>,
     #[serde(default, rename = "tensorzero::deny_unknown_fields")]
     tensorzero_deny_unknown_fields: bool,
+    #[serde(default, rename = "tensorzero::credentials")]
+    tensorzero_credentials: InferenceCredentials,
     #[serde(flatten)]
     unknown_fields: HashMap<String, Value>,
 }
 
+// this implementation ignores tensorzero_credentials and unknown_fields
+impl PartialEq for OpenAICompatibleParams {
+    fn eq(&self, other: &Self) -> bool {
+        self.messages == other.messages
```

Comment:
Could you change this to explicitly name or ignore every field with `let OpenAICompatibleParams { messages, model, frequency_penalty, tensorzero_credentials: _ } = self;`? This will make sure that we get a compiler error if any new fields are later added, so that we remember to manually check them.

Reply:
Never mind - we can actually just remove the `PartialEq` impl entirely. I'll make the change